Assets/Scripts/PlayerController.cs

```diff
@@ -42,48 +42,51 @@ private void Awake()
 
     private void Update()
     {
-        //Input value
-        h = Input.GetAxisRaw("Horizontal");
-        v = Input.GetAxisRaw("Vertical");
-        //Check button up & down
-        bool hDown = Input.GetButtonDown("Horizontal"); //horizontal pressed
-        bool hUp = Input.GetButtonUp("Horizontal"); //horizontal released
-        bool vDown = Input.GetButtonDown("Vertical"); //vertical pressed
-        bool vUp = Input.GetButtonUp("Vertical"); //vertical released
-        //Cross direction movement
-        if (hDown)
+        //disable all of inputs until notification gone
+        if (!BossStageAlter.isWarningOn)
         {
-            isHorizonMove = true;
-        }
-        else if (vDown)
-        {
-            isHorizonMove = false;
-        }
-        else if (hUp || vUp)
-        {
-            isHorizonMove = h != 0;
-        }
-        //Swap the bomba & bamba
-        if (Input.GetKeyDown(KeyCode.E) && bombOnMap == 0)
-        {
-            if (swapSwitch)
+            //Input value
+            h = Input.GetAxisRaw("Horizontal");
+            v = Input.GetAxisRaw("Vertical");
+            //Check button up & down
+            bool hDown = Input.GetButtonDown("Horizontal"); //horizontal pressed
+            bool hUp = Input.GetButtonUp("Horizontal"); //horizontal released
+            bool vDown = Input.GetButtonDown("Vertical"); //vertical pressed

```

Comment:
i like the names of these comments here, but the variable names are unclear. i think you should change the names of the bools to match what you have in the comments, then you won't even need the comments